### DISCUSSION TITLE
Production

### DISCUSSION THREAD

**Discussion by cooperRjp** [markshust/docker-magento#626] [General]
Hi All,

What are your thoughts on using this setup in production for a small store, is it the way to go or would you look at stand alone VMs?

Currently I have a single EC2 instance with ubuntu server running everything, no containers but I'm thinking to use docker instead to separate the services, then perhaps implement Kubernetes if possible so I'm ready to scale. 

Or the other option I've looked at is using a Magento hosting company with a managed service?

I'm finding it hard to find info on Magento open source as Adobe Commerce is confusing things for me with their cloud offering and name changes.

Would just like to get some opinions and guidance?

**Comment by joesken** [comment]
Hi @cooperRjp, just my opinion obviously, I'd strongly recommend a decent Magento hosting company rather than roll your own solution for this type of scenario. Magento gets complicated enough; having to be a sys admin and security expert on top of that is really not worth the hassle.  On top of that updates and upgrades are looked after for you.

YMMV!
